api/wiremock/mappings/get-tax-clearance-success.json

```diff
@@ -0,0 +1,30 @@
+{
+  "request": {
+    "method": "POST",
+    "url": "/tax-clearance",
+    "bodyPatterns": [
+      {
+        "contains": {
+          "taxpayerId": "111-222-333/444",
+          "busName": "Apple Inc.",
+          "repName ": "John Smith",
+          "repId": "f26123fd-54da-4c06-9eb6-ef566c54481a",
+          "agencyName": "New Jersey Economic Development Authority",
+          "corporateNbr": "0100351487",
+          "addressLine1": "1 Main St",
+          "addressLine2": "1st Floor",
+          "city": "Trenton",
+          "state": "NJ",
+          "zip": "08608"
+        }
+      }
+    ]
```

Comment:
I'd like for this to just look to confirm that someone sent all the expected fields and not necessarily validate the information. Right now this requires that someone plugs in this exact information to get a successful response. I just asked the [NJ AI Assistant](https://ai-assistant.nj.gov/) (ChatGPT) to update the code to do this. I think this will work.

I also don't want to require `addressLine2`. If it's provided, great, but we shouldn't require it.

```suggestion
"bodyPatterns": [
      {
        "matchesJsonPath": "$.taxpayerId"
      },
      {
        "matchesJsonPath": "$.busName"
      },
      {
        "matchesJsonPath": "$.repName"
      },
      {
        "matchesJsonPath": "$.repId"
      },
      {
        "matchesJsonPath": "$.agencyName"
      },
      {
        "matchesJsonPath": "$.corporateNbr"
      },
      {
        "matchesJsonPath": "$.addressLine1"
      },
      {
        "matchesJsonPath": "$.city"
      },
      {
        "matchesJsonPath": "$.state"
      },
      {
        "matchesJsonPath": "$.zip"
      }
    ]
```